FirebaseRemoteConfig/Sources/Private/RCNConfigFetch.h

```diff
@@ -48,7 +48,8 @@ typedef void (^RCNConfigFetcherCompletion)(NSData *data, NSURLResponse *response
 /// @param expirationDuration  Expiration duration, in seconds.
 /// @param completionHandler   Callback handler.
```

Comment:
we should briefly explain here why `excludeEtagHeaderForRealtime` is needed

Reply:
Done